google/ggrc-core
# GGRC-4971 Improve performance for Change Log

# Issue description
Research and improve performance of change log

# Steps to test the changes
Launch application server in "launch_ggrc" mode (because in this case memcached will be disabled), and check performance for Change Log tab with and without changes.
Run integration tests 

# Solution description
Decompose load_access_control_list method into several small and replace query with "group by" into several queries with filters, improve performance with generators

# Sanity checklist


# PR Review checklist
